pycytominer/tests/test_feature_select.py

```diff
@@ -73,6 +71,41 @@
 ).reset_index(drop=True)
 
 
+def test_feature_select_noise_removal():
+    """
+    Testing noise_removal feature selection operation
+    """
+    # Set perturbation groups for the test dataframes
```

Comment:
you will also need to write a separate test for when the user specifies a metadata column